src/pages/RFOX/components/Claim/ClaimRow.tsx

```diff
@@ -71,11 +75,23 @@ export const ClaimRow: FC<ClaimRowProps> = ({
       index,
     }
     setConfirmedQuote(claimQuote)
+    // We use this component in two places:
+    // 1. In dashboard `<Claims />`, where there is no widget `<ClaimRoutes />` memory router
+    // 2. In `<ClaimRoutes />`, where there *is* a memory router
+    // Trying to history.push in the first case will result in an invalid route, since there is no ClaimRoutePaths context in <Claims />
+    if (!isClaimWidget) return
```

Comment:
Don't you think that it would better to remove this `history.push` from there and let the parent redirect the UI if needed? As this component can be used in multiple paradigm now, the problem would be kicked by itself by removing the push from there and put it back in `ClaimSelect`

Reply:
Discussed this one out and agreed to pass event handlers/comment out the button for now: e4108e73e2e154fca6fab9143e2f76f1f3d0c169